src/Microsoft.ApplicationInsights/Extensibility/W3C/W3CUtilities.cs

```diff
@@ -12,6 +14,7 @@
     public static class W3CUtilities
```

Comment:
please deperecate this one (or at least all methods except FormatId.. ) and don't bother with missing DiagnosticSource.dll here - it should not be handled here

Reply:
GenerateTraceId() is the only public method. Marked it obsolete. Rest all removed/retained as needed, but are internal only.